### PR TITLE
[MM-58410] Migrate tooltips of 'components/post/post_component' to WithTooltip

### PR DESCRIPTION
**Pull request by ahmadJT**
#### Summary

- [x] Migrate tooltips of 'components/post/post_component' to WithTooltip

#### Ticket Link

Fixes https://github.com/mattermost/mattermost/issues/27153
https://mattermost.atlassian.net/browse/MM-58410

#### Release Note

```release-note

```

### CODE REVIEW

File: webapp/channels/src/components/post/post_component.tsx [hunk 1]
```diff
@@ -593,17 +592,13 @@ const PostComponent = (props: Props): JSX.Element => {
...
                                 {priority}
                                 {post.props && post.props.card &&
-                                    <OverlayTrigger
-                                        delayShow={Constants.OVERLAY_TIME_DELAY}
+                                    <WithTooltip
+                                        id='post_info.info.view_additional_info'
+                                        title={{
+                                            id: 'post_info.info.view_additional_info',
+                                            defaultMessage: 'View additional info',
+                                        }}
```
Comment (M-ZubairAhmed) [inline webapp/channels/src/components/post/post_component.tsx:600]: Please add FormattedMessage component for it's translation to be extractable

**Review by M-ZubairAhmed — CHANGES_REQUESTED** [review]

**Comment by M-ZubairAhmed** [comment]
@ahmadJT can we get this to finish line?

**Review by M-ZubairAhmed — APPROVED** [review]

**Comment by M-ZubairAhmed** [comment]
@Rajat-Dabade can you review this one out

**Review by Rajat-Dabade — APPROVED** [review]

**Review by yasserfaraazkhan — APPROVED** [review]
Hi @M-ZubairAhmed 
can you help in reviewing failing tests ?

I checked the message posts tool tips manually.

**Comment by M-ZubairAhmed** [comment]
@yasserfaraazkhan i checked all the failing tests they are not due to any of the changes in this PR

**Pull request merged**